File: java/배열 만들기 5.java
import java.util.*;

class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        ArrayList<Integer> list = new ArrayList<>();
        for(int i = 0; i < intStrs.length; i++) {
            int element = Integer.parseInt(intStrs[i].substring(s, s+l));
            if(k < element) {
                list.add(element);
            }
        }
        int[] answer = new int[list.size()];
        for(int j = 0; j < answer.length; j++) {
            answer[j] = list.get(j);
        }
        return answer;
    }
}

File: javascript/배열 만들기 5.js
const solution = (intStrs, k, s, l) => {
    let answer = [];
    for(let i of intStrs) {
        let element = parseInt(i.substring(s, s+l));
        if(k < element){
            answer.push(element);
        }
    }
    return answer;
}